### DISCUSSION TITLE
User Profile [Jun 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6006] [Discord Threads]
> 🙋‍♂️ **nomercy**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### currently we are only using email. username is not in use. user profile s a feature in roadmap n we need to redesign a bit first  
#  
> 🙋‍♂️ **nomercy**  
#### I want to work on this feature that is why i was asking those question  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1111545429579018260)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
